README.md

```diff
@@ -18,3 +18,397 @@ We are looking for more hands to join in - or review the task list so that we ca
 See the (draft) [documentation](https://ascii-supply-networks.github.io/dagster-slurm/)
 
 > We are actively looking for contributions to bring this package to life together
+
+
+## developing
+
+```bash
+docker compose up --build -d
+ssh submitter@localhost -p 2222
+# password: submitter
+sinfo
+```
+
+## basic distribution
+
+initial setup
+
+```bash
+curl -fsSL https://pixi.sh/install.sh | sh
+bash
+pixi global install pixi-unpack
+```
+
+environment setup
```

Comment:
What exactly are we setting up here in terms of environment? It would be helpful to understand what the goal here is. It seems like in this section we're distributing the environment to the worker containers?